src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -606,11 +606,15 @@ private int triggerBuild(final AbstractProject<?, ?> triggerProject, final Abstr
         if (manualTrigger != null) {
             final Set<String> downstreamProjectsNames =
                     Sets.newHashSet(Splitter.on(",").trimResults().split(manualTrigger.getDownstreamProjectNames()));
-            if (downstreamProjectsNames.contains(project.getFullName())) {

```

Comment:
2 persons was unable to reproduce https://issues.jenkins-ci.org/browse/JENKINS-26160 . Seems you do something wrong. When you define "project to build" you must enter the full path "Folder/job".